tests/unit/controllers/SessionControllerTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace prime\tests\unit\controllers;
+
+use prime\controllers\SessionController;
+use yii\filters\AccessControl;
+use yii\filters\AccessRule;
+use yii\web\Request;
+use yii\web\User;
+
+/**
+ * @covers \prime\controllers\SessionController
+ */
+final class SessionControllerTest extends ControllerTest
+{
+    public function testSessionRules(): void
+    {
+        $controller = $this->getController(); // new controller instance
+        $this->assertInstanceOf(SessionController::class, $controller); //test instance
+
+        //get access behaviour and test that the rules exist
+        $accessControl = $controller->getBehavior('access');
+        $this->assertInstanceOf(AccessControl::class, $accessControl);
+
+         $rule = $accessControl->rules[0];
+        $this->assertInstanceOf(AccessRule::class, $rule);
+
+        $rule = $accessControl->rules[1];
+        $this->assertInstanceOf(AccessRule::class, $rule);
+
+        #test create action
```

Comment:
The reason that you're adding these comments is that your code is doing multiple things.

The function name (which is the name of the test case) should tell me what is tested.